packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -579,25 +578,11 @@ class PageTransitionsTheme with Diagnosticable {
   Map<TargetPlatform, PageTransitionsBuilder> get builders => _builders;
   final Map<TargetPlatform, PageTransitionsBuilder> _builders;
 
-  /// Delegates to the builder for the current [ThemeData.platform]
-  /// or [FadeUpwardsPageTransitionsBuilder].
-  ///
-  /// [MaterialPageRoute.buildTransitions] delegates to this method.
-  Widget buildTransitions<T>(
```

Comment:
The change makes sense to me. but it would be a breaking change.   cc @HansMuller who may have better idea on how to fix this issue.